docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -62,43 +62,122 @@ and details on the intrapartum and neonatal time periods.
-- Any downstream effects on neonatal or later child outcomes can be modeled with with dummy impact variables if the impacts cannot be properly included yet
+.. _mncnh_portfolio_3.1:
 
-**Wave 4**: Add the neonatal period and related interventions
+3.1 Model Components
+--------------------
+
+Our model will have 3 main "components" that each represent a part of a 
+parent and child dyad's journey. In this model, the simulants (note: simulant 
+here is a parent child dyad) will each run through 3 decision tree based "time 
+steps", where future decisions are based on what the simulant previously experienced. 
+
+For each phase of the model, we will provide inputs needed (state table tracked values), 
+outputs that need to be recorded, and the decision tree. Nodes in the decision trees are 
+all labeled and additional information will be included below.
+
+3.1.1 Wave 1 Model Components
+-----------------------------
+
+**Component 1**: The Pregnancy Model
+
+.. image:: pregnancy_decision_tree_vr.svg
+
+.. list-table:: Pregnancy Decision Tree
+  :widths: 3 15 15
+  :header-rows: 1
+
+  * - ID
+    - Decision Information 
+    - Notes
+  * - 1
+    - XX% of simulants will receive ANC care, based on their propensity 
+    - Several details are still outstanding including: will ANC vary with age, subnational location or upstream factors, will ANC care propensity by correlated with delivery facility propensity
+  * - 2
+    - XX% of simulants will receive AI-assisted ultrasound, XX% will receive standard ultrasound, XX% will receive no ultrasound 
+    - Need to determine if this is random or correlated with other outcomes. Need to find baseline and scenario values.
+  * - 3
+    - All patients are assigned a stated gestational age (GA). Their stated GA will be their real GA +/- an error margin randomly assigned within an interval. For AI assisted ultrasound, the interval is +/- XX weeks, for standard ultrasound XX weeks, for no ultrasound XX weeks. 
+    - Check if distribution for error is random or more likely to be an over/underestimate. 
+  * - 4
+    - XX% of those who received AI-assisted ultrasound will be identified as SGA, XX% with standard ultrasound and XX% with no ultrasound. 
+    - 
+
+
+.. list-table:: Inputs to Pregnancy Decision Tree
+  :widths: 3 15 15
+  :header-rows: 1
+
+  * - Input
+    - Data Source 
+    - Notes
+  * - Age 
+    - GBD and fertility model 
+    - Will be the same population generation as used in nutrition optimization pregnancy model 
+  * - Upstream factors
+    - Likely DHS 
+    - Need to decide what if anything we want to include
+  * - ANC Visit Propensity
+    - Likely DHS 
+    - Need to determine correlation if we want to use it 
+  * - Gestational age at birth
+    - GBD LBWSG
+    - 
+  * - Small for gestational age at birth 
+    - GBD LBWSG
+    - Need to find definition for SGA and determine percent, need to define overlap with gestational age if relevant
```

Comment:
For future reference:

https://en.wikipedia.org/wiki/Small_for_gestational_age

https://www.semanticscholar.org/paper/New-Definition-of-Small-for-Gestational-Age-Based-Gardosi/4c2450962d96043a53479719da252b61ad4baa92